src/controllers/accomodation.controller.js

```diff
@@ -70,6 +70,29 @@ class Accommodation {
     const query = await accomodationServices.bookAccommodation(req.user.id, req.body.accommodationId, req.body.roomId, req.body.departureDate, req.body.checkoutDate);
     return response.successMessage(res, 'Booking was successfully processed', 201, query);
   }
+
+  /**
+   * Like an accommodation
+   * @param {Object} req The request object
+   * @param {Object} res The response object
+   * @returns {Promise} res
+   */
+  static async likeOrUnlike(req, res) {
+    const query = await accomodationServices.likeOrUnlike(req.body.isLike, req.user.id, req.params.accommodationId);
+    const likeOrUnlike = req.body.isLike ? 'Like' : 'Unlike';
+    return response.successMessage(res, `${likeOrUnlike} has successfully done`, 200, query);
+  }
+
+  /**
+   * Has user liked or unliked an accommodation
+   * @param {Object} req The request object
+   * @param {Object} res The response object
+   * @returns {Promise} res
+   */
+  static async hasUserLikeOrUnlike(req, res) {
```

Comment:
@VedaDom may you update this function name to a name that clearly conveys what this function returns?